examples/dbscan/converter.cpp

```diff
@@ -21,17 +21,41 @@
 #include <stdexcept>
 #include <vector>
 
-struct Point
+class Points
 {
-  float _data[3] = {};
-  constexpr Point() noexcept = default;
-  constexpr Point(float x, float y, float z)
-      : _data{x, y, z}
+private:
+  std::vector<std::vector<float>> _data;
+
+public:
+  Points(int dim, int num_points = 0)
```

Comment:
Does it make sense to call the constructor with the default number of points? You can't change that later anyway, can you?

Reply:
OK. You can of course always `push_back`.

Reply:
It still feels a bit sketchy to do that, though. Because you could accidentally end up with the `std:vector`s having different sizes. I guess it's not critical, though.

Reply:
Yes, this is of course for `push_back`, as when reading those CSV files I don't really know how many points there are until I fully read it in.